app/presenters/publishing_api/organisation_presenter.rb

```diff
@@ -90,7 +90,13 @@ def acronym
     end
 
     def summary
-      "#{item.summary}#{organisation_display_name_including_parental_and_child_relationships(item)}"
+      "#{item.summary}#{parent_child_relationships_text}"
+    end
+
+    def parent_child_relationships_text
+      if (item.parent_organisations.any? || item.supporting_bodies.any?)
+        "\n\n#{organisation_display_name_including_parental_and_child_relationships(item)}"
```

Comment:
what's the spacing for?

Reply:
I think is the thing where sometimes there isn't a space between this sentence and the rest of the body? (see below towards the end of the paragraph)

![screen shot 2018-06-18 at 09 45 29](https://user-images.githubusercontent.com/861310/41529307-9a3bbd8a-72e4-11e8-8d51-f59598b6eb2f.png)

Reply:
Ah, I just thought it was weird having new lines in an HTML string.